extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -47,12 +47,18 @@ class AmpStickyAd extends AMP.BaseElement {
     /** @const @private {!Element} */
     this.ad_ = children[0];
 
+    /** @private @const {!Element|null*/
+    this.closeButton_ = null;
+
     /** @private @const {!Viewport} */
     this.viewport_ = this.getViewport();
 
     /** @const @private {!Vsync} */
     this.vsync_ = this.getVsync();
 
+    /** @private @ const {!Function|null} */
+    this.bindedOnClick_ = null;
```

Comment:
Yes, we bind the button only once. I put it here so I can remove listener later. So it's the same question, do we need to remove onlick?